src/api/app/controllers/webui/package_controller.rb

```diff
@@ -7,40 +7,27 @@ class Webui::PackageController < Webui::WebuiController
   include Webui::ManageRelationships
   include BuildLogSupport
 
-  before_action :set_project, only: [:show, :index, :users, :linking_packages, :dependency, :binary, :binaries,
-                                     :requests, :statistics, :commit, :revisions, :submit_request_dialog,
-                                     :branch_diff_info,
-                                     :add_person, :add_group, :rdiff, :save_new,
-                                     :save, :delete_dialog,
-                                     :remove, :add_file, :save_file, :remove_file, :save_person,
-                                     :save_group, :remove_role, :view_file,
-                                     :abort_build, :trigger_rebuild, :trigger_services,
-                                     :wipe_binaries, :buildresult, :rpmlint_result, :rpmlint_log, :meta,
-                                     :save_meta, :attributes, :edit, :files, :binary_download]
-
-  before_action :require_package, only: [:show, :linking_packages, :dependency, :binary, :binaries,
-                                         :requests, :statistics, :commit, :revisions, :submit_request_dialog,
-                                         :branch_diff_info,
-                                         :add_person, :add_group, :rdiff,
-                                         :save, :save_meta, :delete_dialog,
-                                         :remove, :add_file, :save_file, :remove_file, :save_person,
-                                         :save_group, :remove_role, :view_file,
-                                         :abort_build, :trigger_rebuild, :trigger_services,
-                                         :wipe_binaries, :buildresult, :rpmlint_result, :rpmlint_log, :meta,
-                                         :attributes, :edit, :files, :users, :binary_download]
+  before_action :set_project, only: [:show, :index, :users, :linking_packages, :dependency, :binary, :binaries, :requests, :statistics, :revisions,
```

Comment:
`:linking_packages` needs to be removed from all filters (`before_action` and such)

Reply:
@dmarcoux thanks, overlooked it, but I'm currently going through the commits :)

Reply:
@dmarcoux removed it